### PR TITLE
fix(iplb): fix back-link of private network creation

### PR DESCRIPTION
**Pull request by Netnix**
Fix back link next to title so that is links to the previous page
instead of the iplb-home.
Replace oui-back-button by cui-back-link, also remove cui-back-link

### CODE REVIEW

File: client/app/iplb/zone/iplb-zone-delete.html [hunk 1]
```diff
@@ -1,15 +1,12 @@
 <section class="cui-page__content">
-    <cui-back-link data-state="{{ $ctrl.previousState.state }}"
-                   data-state-params="$ctrl.previousState.stateParams"
-                   data-text="'common_back' | translate"></cui-back-link>
-    <cui-page-content-title text="'iplb_zone_delete_title' | translate"></cui-page-content-title>
-    <p data-ng-if="$ctrl.getDeletableZoneCount() === 1" 
-        data-translate="iplb_zone_delete_intro_text_single" 
+    <oui-back-button data-title="{{ :: 'iplb_zone_delete_title' | translate }}"></oui-back-button>
+    <p data-ng-if="$ctrl.getDeletableZoneCount() === 1"
```
Comment (jleveugle) [inline client/app/iplb/zone/iplb-zone-delete.html]: Hey,

I think we should talk about https://angular-translate.github.io/docs/#/guide/14_pluralization ^.^

Comment (jleveugle) [reply client/app/iplb/zone/iplb-zone-delete.html]: or you can use https://docs.angularjs.org/api/ng/directive/ngPluralize too :)

File: client/app/cloud/project/storage/storage-add/storage-add.html [hunk 2]
```diff
@@ -3,9 +3,7 @@
     <div data-ng-include="'app/cloud/project/storage/storage-header.html'"></div>
 
     <section class="cui-page__content">
-        <cui-back-link data-state="iaas.pci-project.storage" data-text="'storage_add_button_return' |translate"></cui-back-link>
-
-        <cui-page-content-title data-text="contentTitle"></cui-page-content-title>
+        <oui-back-button data-title="{{ :: contentTitle }}"></oui-back-button>
```
Comment (FredericEspiau) [inline client/app/cloud/project/storage/storage-add/storage-add.html]: Does it actually do anything without `href` or `on-click` ?

Comment (Netnix) [reply client/app/cloud/project/storage/storage-add/storage-add.html]: Yes, it does the default behavior, which is: $window.history.back()

**Pull request merged**